samples/Ecs/AudioEcsCAPI/main.c

```diff
@@ -0,0 +1,287 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#define NOVELRT_C_API 1
+#include <NovelRT.Interop/Ecs/Audio/NrtEcsAudio.h>
+#include <NovelRT.h>
```

Comment:
Shouldn't you only need NovelRT.h here since you're defining the C API flag for it? What am I misunderstanding?

Reply:
Probably  - looks like missing headers for NovelRT.h